cohortextractor2/dsl.py

```diff
@@ -59,10 +59,17 @@
 
 
 class Cohort:
-    """Represents the cohort of patients in a study."""
+    """
+    Represents the cohort of patients in the defined study.
+    """
 
     def set_population(self, population: PatientSeries) -> None:
-        """Set the population variable for this cohort."""
+        """
+        This sets the population that are included within the Cohort
+
+        Args:
+            population: Patients who are included within the Cohort
```

Comment:
This isn't strictly a series containing the patients. It's a boolean series and whether the boolean is true or false dictates whether that patient is included.